multimedia/minisatip/files/minisatip.init

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh /etc/rc.common
+
+USE_PROCD=1
+
+start_service() {
+  procd_open_instance [minisatip]
```

Comment:
Please consider removing the square brackets in the instance name.  It's purely decorative but not conventional